exporter/awsxrayexporter/internal/translator/aws.go

```diff
@@ -167,7 +168,15 @@ func makeAws(attributes map[string]pcommon.Value, resource pcommon.Resource, log
 		queueURL = value.Str()
 	}
 	if value, ok := attributes[conventions.AttributeAWSDynamoDBTableNames]; ok {
-		tableName = value.Str()
+		if value.Slice().Len() == 1 {
+			tableName = value.Slice().At(0).Str()
+		} else if value.Slice().Len() > 1 {
```

Comment:
XRay console treats `tableName` and `tableNames` differently. Depending on which is present in the trace, it will create a different type of node. This is why this distinction is needed.
![image](https://user-images.githubusercontent.com/107072447/228639398-25eeb729-d2e5-448f-aab9-edc54ad64263.png)